packages/atcommons/include/atcommons/enroll_namespace.h

```diff
@@ -0,0 +1,64 @@
+#ifndef ENROLL_NAMESPACE_STRUCT_H
+#define ENROLL_NAMESPACE_STRUCT_H
+
+#include <stddef.h>
+
+typedef struct {
+  char *name;
+  char *access;
+} enroll_namespace_t;
+
+typedef struct {
+  int length;
```

Comment:
use size_t

Reply:
change made as part of commit: [ac675cf](https://github.com/atsign-foundation/at_c/pull/423/commits/ac675cf72fe01483c980613bd39e186792bc865d)